android/kotlin-central-login-oidc/app/src/main/java/com/example/app/Error.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2023 ForgeRock. All rights reserved.
```

Comment:
Another thing to confirm regarding the copyright note - i.e. shall we update this to 2024...